hack/generate-manifest.sh

```diff
@@ -390,6 +402,14 @@ if $WHEREABOUTS; then
     HELM_VALUES+=("whereabouts.enable=true")
 fi
 
+if $ENABLE_NODEPORT; then
+    HELM_VALUES+=("nodeport.enable=true" "nodeport.port=32767")
```

Comment:
Better to pass a port number with --enable-nodeport

Reply:
Sure. I will update it.